src/components/shared/useState/index.tsx

```diff
@@ -1,37 +1,37 @@
-import React, { useState } from "react";
+import React, { useState } from 'react'
 
-import styles from "./state.module.scss";
+import styles from './state.module.scss'
 
 const computed = () => {
-  return Math.trunc(Math.random() * 20);
-};
+  return Math.trunc(Math.random() * 20)
+}
 
 export default function UseState(): JSX.Element {
-  const [count, setCount] = useState<number>(computed());
```

Comment:
why did you remove `computed()` and put `0` , you need to create a component from the beginning to work there,

Reply:
vortev mez petqa naxnakan arjeq tanq mianqamic 0 em tvel